chef_master/source/install_chef_air_gap.rst

```diff
@@ -0,0 +1,602 @@
+
+This guide will show you how to run a fully functional Chef environment within an `air-gapped <https://en.wikipedia.org/wiki/Air_gap_(networking)>`__ network.
+
+Prerequisites
+=====================================================
+
+Since a variety of different practices are used to create an air-gapped network, this guide focuses solely on the implementation of Chef software - as such, it makes the following assumptions:
+
+* You have a way to get packages to your air-gapped machines
+* Machines on your air-gapped network are able to resolve each other via DNS
+* A server's Fully Qualified Domain Name (FQDN) is the name that will be used by other servers to access it
+* You have a private Ruby gem mirror to supply gems as needed
+* You have an artifact store for file downloads. At a minimum, it should have the following packages available:
+    * Chef DK
+    * Chef client
+    * Chef Supermarket
+    * An `install script </#create-an-install-script>`__ for Chef client
+
+Required cookbooks
+-----------------------------------------------------
+This guide will link to the required cookbooks for each piece of software in that software's respective section, but this is a full list of the cookbooks required to complete the entire guide:
+
+For Chef Supermarket:
+* `Supermarket-omnibus-cookbook <https://supermarket.chef.io/cookbooks/supermarket-omnibus-cookbook>`_
+* `Chef-ingredient <https://supermarket.chef.io/cookbooks/chef-ingredient>`_
+* `hostsfile <https://supermarket.chef.io/cookbooks/hostsfile>`_ 
+
+For Chef Automate:
+* `delivery-sugar <https://supermarket.chef.io/cookbooks/delivery-sugar>`_
+* `delivery-truck <https://supermarket.chef.io/cookbooks/delivery-truck>`_
+
+Create an install script
+-----------------------------------------------------
+An install script is used to install Chef client when bootstrapping a new node. It simply pulls the Chef client package from your artifact store, and then installs it. For example, on Debian-based Linux systems, it would look similar to this:
+
+.. code-block:: bash
+
+   #!/bin/bash
+
+   cd /tmp/
+   wget http://packages.example.com/chef_13.2.20-1_amd64.deb
+   dpkg -i chef_13.2.20-1_amd64.deb
+
+The install script should be accessible from your artifact store.
+
+Chef server
+=====================================================
+In this section you'll create install the Chef server, and create your organization and user.  Note that in order to configure Supermarket later in this guide, you will need a user that is a member of the ``admins`` group.
+
+.. note:: If you intend to use Chef Automate, create the ``delivery`` user and add it to your organization during this step.
+
+#. Download the package from https://downloads.chef.io/chef-server/.
+#. Upload the package to the machine that will run the Chef server, and then record its location on the file system. The rest of these steps assume this location is in the ``/tmp`` directory.
+
+#. .. tag install_chef_server_install_package
```

Comment:
Because this guide assumes someone might be reading it via pdf within an air-gapped network, it pulls in a lot of install information from other guides. As a result, it's quite long for a procedural doc - and I'm wary of making it any longer. 

We're going to have to get a sense of what add-ons and features are most in demand for users in an air-gapped environment (assuming that the offline aspect effects the setup process somehow) and then potentially create a subset of documentation for air-gapped networks.